easybuild/framework/easyconfig/format/format.py

```diff
@@ -65,9 +65,11 @@
     ['source_urls', 'sources', 'patches', 'checksums'],
     DEPENDENCY_PARAMETERS + ['multi_deps'],
     ['osdependencies'],
-    ['preconfigopts', 'configopts'],
-    ['prebuildopts', 'buildopts'],
-    ['preinstallopts', 'installopts'],
+    ['preconfigcmds', 'preconfigopts', 'configure_cmd', 'configopts'],
+    ['prebuildcmds', 'prebuildopts', 'build_cmd', 'buildopts'],
```

Comment:
make sure you take into account naming policy here (cfr. https://github.com/easybuilders/easybuild-framework/issues/4464)

Reply:
done in https://github.com/easybuilders/easybuild-framework/pull/4531/commits/38341704c415326fb0c2e175b1b5566ba3234143